scripts/visualization/opf_visualizer.js

```diff
@@ -331,6 +331,20 @@ angular.module('app').controller('AppCtrl', ['$scope', '$timeout', function($sco
     }
   };
 
+  // this function highlights a range in the graph
+  var highlightPoint = function(canvas, area, g) {
+              var bottom_left = g.toDomCoords(20, -20);
+              var top_right = g.toDomCoords(40, +20);
```

Comment:
@jefffohl is the `g.toDomCoords()` correct use here? Now I'm able to draw absolutely-positioned `fillRect(10,0,100,100)`; but when I try to make it relative to the graph (so it moves with it during pan/zoom) - the problem is the `bottom_left` contains some crazy value (`(-246564.xxxxx; 300.xxxx)`)